src/main/java/shon/Parser.java

```diff
@@ -85,15 +85,15 @@ private static Action getAction(String input) throws CommandException {
      * is not a number.
      */
     private static int getIdx(String input) throws ParameterException {
-        String[] split = input.split(" ");
-        if (split.length == 1) {
+        String[] parameters = input.split(" ");
```

Comment:
Very good usage of variable names, a lot of people I have seen use names like `arr` but this is better for code readability!